README.md

```diff
@@ -18,19 +18,36 @@ modules written for V8.
 A rebuild of node and native addon modules with ChakraCore is required for this
 to work.
 
-### Issues
+### Time Travel Debugging 
+Time-Travel debugging is an exciting innovation in Node.js debugging being developed in Node-ChakraCore supported by [VSCode](https://github.com/microsoft/vscode/). We are developing in the open and want to share our progress to get feedback, bug reports, functionality requests, and pull-requests from the community. 
 
-Please report all issues related to Node-ChakraCore on this separate
-[issue page] (https://github.com/nodejs/node-chakracore/issues).
+Check out this [Time-Travel Debugging](https://github.com/nodejs/node-chakracore/blob/xplat/TTD-README.md) page to learn how to get started with TTD on Node-ChakraCore with VSCode.
+
+### Node.js API (NAPI)
+ChakraCore is also supporting the advancement of [ABI-Stable-Node](https://github.com/nodejs/abi-stable-node) project also called NAPI. The compatibility of native add-on Node module ecosystem is going to greatly improve across different node versions and also different Node VMs.
```

Comment:
Nit: , before also called. Also, would the second sentence be better if stated as a goal? "The goal of this project is to ease the lives of native add-ons developers by providing Node a stable ABI that module authors can target, thus improving compatibility across disparate Node versions?